server/network/aoprotocol.py

```diff
@@ -341,29 +342,28 @@ def net_cmd_ms(self, args):
             return
 
         target_area = []
-
-        if self.validate_net_cmd(args, self.ArgType.STR,
-                                 self.ArgType.STR_OR_EMPTY, self.ArgType.STR,
-                                 self.ArgType.STR, self.ArgType.STR,
-                                 self.ArgType.STR, self.ArgType.STR,
-                                 self.ArgType.INT, self.ArgType.INT,
-                                 self.ArgType.INT, self.ArgType.INT,
-                                 self.ArgType.INT, self.ArgType.INT,
-                                 self.ArgType.INT, self.ArgType.INT):
+        if self.validate_net_cmd(args, self.ArgType.STR, # msg_type
+                                 self.ArgType.STR_OR_EMPTY, self.ArgType.STR,#pre, folder
+                                 self.ArgType.STR, self.ArgType.STR,# anim, text
+                                 self.ArgType.STR, self.ArgType.STR,#pos, sfx
+                                 self.ArgType.INT, self.ArgType.INT,#anim_type, cid
+                                 self.ArgType.INT, self.ArgType.INT_OR_STR,#sfx_delay, button
+                                 self.ArgType.INT, self.ArgType.INT,#evidence, flip
+                                 self.ArgType.INT, self.ArgType.INT):#ding, color
             # Pre-2.6 validation monstrosity.
             msg_type, pre, folder, anim, text, pos, sfx, anim_type, cid, sfx_delay, button, evidence, flip, ding, color = args
             showname = ""
             charid_pair = -1
             offset_pair = 0
             nonint_pre = 0
         elif self.validate_net_cmd(
-                args, self.ArgType.STR, self.ArgType.STR_OR_EMPTY,
-                self.ArgType.STR, self.ArgType.STR, self.ArgType.STR,
-                self.ArgType.STR, self.ArgType.STR, self.ArgType.INT,
-                self.ArgType.INT, self.ArgType.INT, self.ArgType.INT,
-                self.ArgType.INT, self.ArgType.INT, self.ArgType.INT,
-                self.ArgType.INT, self.ArgType.STR_OR_EMPTY, self.ArgType.INT,
-                self.ArgType.INT, self.ArgType.INT):
+                args, self.ArgType.STR, self.ArgType.STR_OR_EMPTY,#msg_type, pre
+                self.ArgType.STR, self.ArgType.STR, self.ArgType.STR,#folder, anim, text
+                self.ArgType.STR, self.ArgType.STR, self.ArgType.INT,# pos, sfx, anim_type
+                self.ArgType.INT, self.ArgType.INT, self.ArgType.INT_OR_STR,#cid, sfx_delay, button
+                self.ArgType.INT, self.ArgType.INT, self.ArgType.INT,#evidence, flip, ding
+                self.ArgType.INT, self.ArgType.STR_OR_EMPTY, self.ArgType.INT, #color, showname, charid_pair
+                self.ArgType.INT, self.ArgType.INT): #offset_pair, nonint_pre
```

Comment:
Please leave a space before and after the `#`.

Reply:
Ok forgot to maintain the artistic standard.